Project.toml

```diff
@@ -3,11 +3,15 @@ uuid = "b790e538-3052-4cb9-9f1f-e05859a455f5"
 authors = ["Siddharth Lal"]
 version = "0.1.0"
 
+[deps]
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+
 [compat]
 julia = "1"
 
 [extras]
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
```

Comment:
You don't need to repeat the dependency here if it's already in the deps section. Remove it also from the test target below